toolchain/check/decl_name_stack.cpp

```diff
@@ -76,11 +76,11 @@ auto DeclNameStack::LookupOrAddName(NameContext name_context,
           if (!inst.Is<SemIR::Namespace>()) {
             // TODO: Point at the declaration for the scoped entity.
             CARBON_DIAGNOSTIC(
-                QualifiedDeclOutsideScopeEntity, Error,
+                QualifiedNameOutsideScopeEntity, Error,
                 "Out-of-line declaration requires a declaration in "
                 "scoped entity.");
             context_->emitter().Emit(name_context.parse_node,
-                                     QualifiedDeclOutsideScopeEntity);
+                                     QualifiedNameOutsideScopeEntity);
```

Comment:
These renames lose a little bit of information. These *are* declarations, but of a qualified name. Maybe `DeclaringQualifiedName...` would be a more clear name?

But given the text "qualified declaration outside scope entity" reads about right to me, so maybe this one doesn't need the rename?

Reply:
Done.